homeassistant/components/nanoleaf/light.py

```diff
@@ -194,7 +194,8 @@ async def async_update(self) -> None:
             _LOGGER.info("Fetching %s data recovered", self.name)
         self._attr_available = True
 
-    async def async_handle_update(self) -> None:
+    @callback
+    def handle_update(self) -> None:
```

Comment:
A callback is still async safe :)

```suggestion
    def async_handle_update(self) -> None:
```

Reply:
:upside_down_face: